Library/Homebrew/list.sh

```diff
@@ -9,7 +9,7 @@ homebrew-list() {
     *) ;;
   esac
 
-  local ls_args=()
+  local ls_args=(-C)
```

Comment:
Just noting this may not work as expected unless `COLUMNS` has been set as environment variable. This isn't guaranteed to be set. For portability, would need some `stty` or `tput` workaround.

Reply:
I've fixed this to only apply to a TTY so that fixes your first example there to return just `actionlint`. Does that cover the rest?

Reply:
There is still the situation of this being `COLUMN` environment variable dependent and not portable across user setups. My setup doesn't include the variable, e.g.
```console
❯ zsh

❯ env | rg COLUMNS

❯ echo $COLUMNS
120

❯ tput cols
120
```
```console
❯ bash

❯ env | rg COLUMNS

❯ echo $COLUMNS
120

❯ tput cols
120
```

Perhaps doing something like `COLUMNS=${COLUMNS:-$(tput cols)} ls` to fall back to [`tput`](https://pubs.opengroup.org/onlinepubs/9799919799/utilities/tput.html)

Reply:
Fast path:
```
❯ brew list
==> Formulae
actionlint			libtiff
aom				libtool
...
```

Slow path:
```
❯ brew ls
==> Formulae
actionlint			gron				libunibreak			pixman
aom				harfbuzz			libunistring			pkg-config
...
```

Reply:
Ah ok, I understand what you're saying. I think I've fixed that now.